packages/claim-cli/src/index.ts

```diff
@@ -0,0 +1 @@
+export { run } from '@oclif/core';
```

Comment:
Since there is only one command, we can make it as single command CLI
https://oclif.io/docs/single_command_cli